### PR TITLE
Feature/add readme

### PR DESCRIPTION
**Pull request by rishabhmalik759**

### CODE REVIEW

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- Notes (includes, your reasoning behind the problem or steps you tried to solve the bug)
+
+We _love_ thorough bug reports so that we can act fast on them.
+
+## Use a Consistent Coding Style
+
+> Reference [Facebook's Guidelines](https://github.com/facebook/draft-js/blob/a9316a723f9e918afde44dea68b5f9f39b7d9b00/CONTRIBUTING.md)
+
+- 2 spaces for indentation rather than tabs
+- 80 character line length
```
Comment (Foxhoundn) [inline CONTRIBUTING.MD:57]: 100

File: CONTRIBUTING.MD [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## Any contributions you make will be under the MIT Software License
+
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's issues
+
+We use GitHub issues to track public bugs. Report a bug by opening a new issue on <code>github.com/qoretechnologies/{Repository-Name}/issues</code> it's that easy!
```
Comment (Foxhoundn) [inline CONTRIBUTING.MD:36]: replace repository name with the name

File: CONTRIBUTING.MD [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+> We Use [Github Flow](https://guides.github.com/introduction/flow/index.html) for code contributions
+
+## How to contribute
+
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `develop`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
```
Comment (Foxhoundn) [inline CONTRIBUTING.MD:25]: There is no documentation for Qorus UI

File: CONTRIBUTING.MD [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+4. Ensure the test suite passes.
+5. Make sure your code lints.
```
Comment (Foxhoundn) [inline CONTRIBUTING.MD:27]: No linters either

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
+# Reporting and Fixing Security Issues
+
+Please do not open GitHub issues or pull requests - this makes the problem immediately visible to everyone, including malicious actors. Security issues in Qorus Integration Engine UI can be safely reported via email <b>security@qoretechnologies.com</b>
```
Comment (Foxhoundn) [inline SECURITY.md:4]: Does this email exist?

File: README.md [hunk 6]
```diff
@@ -1,46 +1,36 @@
...
 
-### `yarn test`
+If you discover a security vulnerability in Qorus-Toolkit, please see [Security Policies and Procedures]().
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Contributors
 
-### `yarn build`
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
```
Comment (Foxhoundn) [inline README.md:23]: Remove this sentence from this repo.

**Review by Foxhoundn — CHANGES_REQUESTED** [review]
Please change the logo to some kind of a gradient / shade of the Qorus purple instead of the green.

File: SECURITY.md (hunk 5 above)
Comment (rishabhmalik759) [reply SECURITY.md:4]: no we can change it or make a new one

**Review by Foxhoundn — APPROVED** [review]

**Review by Foxhoundn — CHANGES_REQUESTED** [review]
Please increase the version in package.json

**Review by Foxhoundn — APPROVED** [review]

**Pull request merged**